class-work/src/task/menu_items/Replace.java

```diff
@@ -1,4 +1,26 @@
 package task.menu_items;
 
+import java.util.Scanner;
+
 public class Replace {
+
+    /*      5. Replace (Написать программу, которая заменяет в исходной строке все буквы а на @.
+            Буквы о на ноль. Выводит новую строку на экран.
+            Добавить обработку длины строки, максимальная длина 30 символов.
+            Строка может состоять из нескольких слов и любых символов)*/
+
+    Scanner scanner = new Scanner(System.in);
+
+    public void getStringWithReplacedChars() {
```

Comment:
if method gets something, then return type should be defined with specific data type (not void) and it should be returned.